modules/ROOT/pages/security/configuration.adoc

```diff
@@ -62,14 +80,66 @@ new Neo4jGraphQL({
 });
 ----
 
-[[authentication-and-authorization-jwt]]
-== JWT
+==== Passing in encoded JWTs
+
+// This was at the end of the file, I thought it could be moved here instead. What about decoded JWTs?
+
+To pass in an encoded JWT, use the token field of the context.
+When using Apollo Server, extract the authorization header into the token property of the context:
+
+[source, javascript, indent=0]
+----
+const server = new ApolloServer({
+    schema,
+});
+
+await startStandaloneServer(server, {
+    context: async ({ req }) => ({ token: req.headers.authorization }),
+});
+----
+
+For example, a HTTP request with the following `authorization` header should look like this:
+
+[source]
+----
+POST / HTTP/1.1
+authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyLCJyb2xlcyI6WyJ1c2VyX2FkbWluIiwicG9zdF9hZG1pbiIsImdyb3VwX2FkbWluIl19.IY0LWqgHcjEtOsOw60mqKazhuRFKroSXFQkpCtWpgQI
+content-type: application/json
+----
+
+Alternatively, you can pass a key `jwt` of type `JwtPayload` into the context, which has the following definition:
+
+[source, typescript, indent=0]
+----
+// standard claims https://datatracker.ietf.org/doc/html/rfc7519#section-4.1
+interface JwtPayload {
+    [key: string]: any;
+    iss?: string | undefined;
+    sub?: string | undefined;
+    aud?: string | string[] | undefined;
+    exp?: number | undefined;
+    nbf?: number | undefined;
+    iat?: number | undefined;
+    jti?: string | undefined;
+}
+----
+
+[WARNING]
+Do not pass in the header or the signature.
+
+=== Decoded JWTs
+
+// What could be added here?
```

Comment:
^

Reply:
A decoded JWT is passed in to the context in a similar way that an encoded JWT is, but instead of using `token`, we use `jwt`, like this where the value of the jwt is passed in by code. `customImplementation` should be thought of as a placeholder for whatever the programmer would actually write.

```ts
const jwt = customImplementation();

const { url } = await startStandaloneServer(server, {
    listen: { port: 4000 },
    context: async ({ req }) => ({
        jwt: jwt,
    }),
});
```

Using `jwt` instead of `token` in the context informs the Neo4jGraphQL library that it doesn't need to decode it.

Reply:
adapted, thanks